client/src/Components/Input/DropDown/index.tsx

```diff
@@ -0,0 +1,173 @@
+                            strokeWidth='1.5'
+                            strokeLinecap='round'
+                            strokeLinejoin='round'
+                          />
+                        </svg>
+                      </span>
+                    </Listbox.Button>
+                  </span>
+
+                  {/* RESPONSIBLE FOR THE 'DROPPING DOWN' OF THE MENU */}
+                  <Transition
+                    show={open}
+                    leave='transition ease-in duration-100'
+                    leaveFrom='opacity-100'
+                    leaveTo='opacity-0'
+                    className='absolute mt-1 w-full rounded-md bg-white shadow-lg'
+                  >
+                    <Listbox.Options
+                      static
+                      className='max-h-60 rounded-md py-1 text-base leading-6 shadow-xs overflow-auto focus:outline-none sm:text-sm sm:leading-5'
+                    >
+                      {/* MAPS OPTIONS IN DROPDOWN MENU */}
+                      {options.map(option => (
+                        <Listbox.Option
+                          key={option}
+                          value={option}
+                          disabled={disabled}
+                        >
+                          {({ selected, active }) => (
+                            <div
+                              className={`${
+                                active ? 'text-white bg-orange' : 'text-black'
+                              } cursor-default select-none relative py-2 pl-8 pr-4`}
+                            >
+                              <span
+                                className={`${
+                                  selected ? 'font-semibold' : 'font-normal'
+                                } block truncate`}
+                              >
+                                {option}
+                              </span>
+                              {selected && (
+                                <span
+                                  className={`${
+                                    active ? 'text-white' : 'text-blue-600'
+                                  } absolute inset-y-0 left-0 flex items-center pl-1.5`}
+                                >
+                                  <svg
+                                    className='h-5 w-5'
+                                    xmlns='http://www.w3.org/2000/svg'
+                                    viewBox='0 0 20 20'
+                                    fill='currentColor'
+                                  >
+                                    <path
+                                      fillRule='evenodd'
+                                      d='M16.707 5.293a1 1 0 010 1.414l-8 8a1 1 0 01-1.414 0l-4-4a1 1 0 011.414-1.414L8 12.586l7.293-7.293a1 1 0 011.414 0z'
+                                      clipRule='evenodd'
+                                    />
+                                  </svg>
```

Comment:
I would recommend moving this svg into its own component so its easier to understand what the svg is representing